pkg/rpaas/client/client.go

```diff
@@ -301,3 +301,37 @@ func encodeCertKey(service, instance, certificate, key, destName string) (string
 
 	return body.String(), writer.Boundary(), nil
 }
+
+func (c *RpaasClient) Update(ctx context.Context, service, instance, plan, flavor string) error {
```

Comment:
We need add more fields here such as: description, team owner, tags. Creating a struct to group these parameters instead of passing directly as method parameter may looks a better idea.